src/App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './App.css';
+import {Profile} from './profile/profile'
+
+
+function App() {
+  const styleObject={ border:" 2px solid black" ,};
+  function handleName(name){
+      alert(name);
+  }
+  return (
+    <>
+    <Profile style={styleObject} fullName="Andrew" bio="" profession="" alertName={handleName} >
+      <img  style={styleObject} src="images/img1.png" alt="profileImage" />
```

Comment:
`alt` attributes should provide descriptive alternative of what an image includes in a readable human format.